services/content-unification/unification.py

```diff
@@ -137,3 +141,255 @@ def delete_from_graph(subjects, graph):
     return query_string
 
 
+def remove_vocab_concepts(vocab_uuid: str, graph: str):
+    query_template = Template("""
+PREFIX void: <http://rdfs.org/ns/void#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+WITH $graph
+DELETE {
+    ?concept ?conceptPred ?conceptObj .
+}
+WHERE {
+    ?vocabMeta a ext:VocabularyMeta ;
+                 mu:uuid $vocab_uuid .
+    ?vocabMeta ?vocabMetaPred ?vocabMetaObj .
+    ?vocabMeta ext:sourceDataset ?sourceDataset .
+    ?concept dct:source ?sourceDataset .
+    ?concept ?conceptPred ?conceptObj .
+}
+    """)
+    query_string = query_template.substitute(
+        graph=sparql_escape_uri(graph),
+        vocab_uuid=sparql_escape_string(vocab_uuid),
+    )
+    return query_string
+
+def remove_vocab_data_dumps(vocab_uuid: str, graph: str):
+    query_template = Template("""
+PREFIX void: <http://rdfs.org/ns/void#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>
+
+WITH $graph
+DELETE {
+    ?dataDump ?dataDumpPred ?dataDumpObj .
+    ?dataSource ?dataSourcePred ?dataSourceObj .
+    ?sourceDataset void:dataDump ?dataDump .
+}
+WHERE {
+    ?vocab a ext:VocabularyMeta ;
+           mu:uuid $vocab_uuid .
+    ?vocab ext:sourceDataset ?sourceDataset .
+    ?sourceDataset void:dataDump ?dataDump .
+    ?dataDump ?dataDumpPred ?dataDumpObj .
+    ?dataSource nie:dataSource ?dataDump .
+    ?dataSource ?dataSourcePred ?dataSourceObj .
+}
+    """)
+    query_string = query_template.substitute(
+        graph=sparql_escape_uri(graph),
+        vocab_uuid=sparql_escape_string(vocab_uuid),
+    )
+    return query_string
+
+def remove_vocab_source_datasets(vocab_uuid: str, graph: str):
+    query_template = Template("""
+PREFIX void: <http://rdfs.org/ns/void#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+WITH $graph
+DELETE {
+    ?sourceDataset ?sourceDatasetPred ?sourceDatasetObj
```

Comment:
Om consequent te zijn zou ik net zoals in de query  hierboven ook `?vocab ext:sourceDataset ?sourceDataset .` meenemen bij het verwijderen.